lib/package-managers/abstract.package-manager.ts

```diff
@@ -11,7 +11,7 @@ import { ProjectDependency } from './project.dependency';
 export abstract class AbstractPackageManager {
   constructor(protected runner: AbstractRunner) {}
 
-  public async install(directory: string) {
+  public async install(directory: string, selectedPackageManager: string) {
```

Comment:
maybe just `packageManager: string` (I think it is pretty self-describable in this context)

Reply:
Okay, I will update it